lib/chatMessage/mutation.py

```diff
@@ -0,0 +1,32 @@
+import graphene
+
+from bson import ObjectId
+
+from datetime import datetime
+
+from .type import ChatMessageType
+from .model import ChatMessage
+
+
+class PostChatMessage(graphene.Mutation):
+    chat_message = graphene.Field(ChatMessageType)
+
+    class Arguments:
+        message = graphene.String(required=True)
+        chat_id = graphene.String(required=True)
+
+    def mutate(root, info, chat_id, **args):
+        token = getattr(info.context, "token", None)
+
+        if token is None:
+            return {}
```

Comment:
Return what we would return if it was a success, so a PostChatMessage() or something like that